frontend/src/HelperFunctions/authenticateWithOpenId.js

```diff
@@ -1,5 +1,4 @@
 import axios from 'axios';
-import { useNavigate } from 'react-router-dom'
 
 export default async function authenticateWithOpenId(responseDetails) {
   const baseUrl = localStorage.getItem('homeServer');
```

Comment:
I still wonder if, structually, this would be better off as a `MatrixClient` API class rather than several helper functions. It would help seperate state (`localStorage`) from logic (your requests)`.